.circleci/config.yml

```diff
@@ -0,0 +1,70 @@
+version: 2
+jobs:
+  build:
+    parallelism: 4
```

Comment:
I am not sure if this is going to have the behavior we want. It is going to try and run each step of this job in parallel. See the docs [here](https://circleci.com/docs/2.0/parallelism-faster-jobs/) and [here](https://circleci.com/docs/2.0/configuration-reference/#build).